### PR TITLE
Add tests for module classes and addc command

### LINKED ISSUE #123: Add test cases for `UniqueModuleClassList` and `addc` command

### PR DESCRIPTION
**Pull request by njxue**
1. `TypicalModuleClasses` and `TypicalSessions` contain objects that can be used throughout test cases
2. `ModuleClassBuilder` can be used to create `ModuleClass` objects for module classes in `TypicalModuleClasses`

### CODE REVIEW

File: src/test/data/JsonSerializableTaAssistTest/invalidStudentTaAssist.json [hunk 1]
```diff
@@ -5,5 +5,11 @@
...
     "address": "4th street"
   } ],
-  "moduleClasses" : []
+  "moduleClasses" : [{
+    "name": "CS1101S",
+    "sessions": []
+  }, {
+    "name": "CS1231S",
+    "sessions": []
+  }]
```
Comment (RezwanArefin01) [inline src/test/data/JsonSerializableTaAssistTest/invalidStudentTaAssist.json:14]: This is an invalid files, so there was no need to do this. :joy:

File: src/test/java/seedu/taassist/logic/commands/AddcCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        AddcCommand addcCommand = new AddcCommand(validModuleClass);
+        ModelStubWithModuleClass modelStub = new ModelStubWithModuleClass(validModuleClass);
+
+        assertThrows(CommandException.class, AddcCommand.MESSAGE_DUPLICATE_MODULE_CLASS, () ->
+                addcCommand.execute(modelStub));
+    }
+
+    @Test
+    public void equals() {
+        AddcCommand addCS1101SCommand = new AddcCommand(CS1101S);
```
Comment (RezwanArefin01) [inline src/test/java/seedu/taassist/logic/commands/AddcCommandTest.java]: Should `addCS1101SCommand` be `addCs1101sCommand` instead? as per coding standard.

File: src/test/java/seedu/taassist/model/moduleclass/UniqueModuleClassListTest.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.taassist.model.moduleclass.exceptions.DuplicateModuleClassException;
+import seedu.taassist.model.moduleclass.exceptions.ModuleClassNotFoundException;
+
+public class UniqueModuleClassListTest {
```
Comment (RezwanArefin01) [inline src/test/java/seedu/taassist/model/moduleclass/UniqueModuleClassListTest.java]: PR #130 removes the Unique* classes.

**Review by RezwanArefin01 — CHANGES_REQUESTED** [review]
Looks good. One or two minor nits.

File: src/test/data/JsonSerializableTaAssistTest/invalidStudentTaAssist.json (hunk 1 above)
Comment (njxue) [reply src/test/data/JsonSerializableTaAssistTest/invalidStudentTaAssist.json:14]: This file is used to test an invalid student field (the invalid email) and not the json file, I think. One of the test cases fails (with read json file error) if I do not change it

File: src/test/java/seedu/taassist/logic/commands/AddcCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+        assertThrows(CommandException.class, AddcCommand.MESSAGE_DUPLICATE_MODULE_CLASS, () ->
+                addcCommand.execute(modelStub));
+    }
+
+    @Test
+    public void equals() {
+        AddcCommand addCs1101sCommand = new AddcCommand(CS1101S);
+        AddcCommand addCs1101sCommandCopy = new AddcCommand(CS1101S);
+        AddcCommand addCS1231SCommand = new AddcCommand(CS1231S);
```
Comment (RezwanArefin01) [inline src/test/java/seedu/taassist/logic/commands/AddcCommandTest.java]: Leftover coding standard issue.

**Review by RezwanArefin01 — CHANGES_REQUESTED** [review]
One minor coding standard issue left over.

**Review by RezwanArefin01 — APPROVED** [review]
LGTM!

**Pull request merged**